plugins/chat/app/services/chat/update_thread_title_prompt.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
```

Comment:
Should change service name too? Maybe something more explicit:

mark_thread_title_prompt_seen ?